flows/patterns/patterns.md

```diff
@@ -2,9 +2,25 @@
 
 This page demonstrates how to accomplish some common tasks in the flows system.
 
-## Build an HTML email body with query results
+## Send Query Results via Email
 
-A very common use-case for flows is to run one or more queries, check if there were results, and send an alert email if so. The following flow implements the basics:
+One of the simplest but most useful things you can do in a flow is to periodically send the results of a query via email. It only takes two nodes to do this:
+
+![](simple-email1.png)
+
+The [Run Query](/flows/nodes/runquery) node is configured to execute a search, in this case `tag=gravwell syslog Appname | stats count by Appname | table`.
+
+The [Email](/flows/nodes/email) node is then configured to use `search.Results` as the *Body* of the email. Gravwell will automatically format the search results as HTML when sending.
+
+![](simple-email2.png)
+
+The resulting email is shown below:
+
+![](simple-email3.png)
+
+## Build an arbitrary HTML email body
+
+Although you can send the results of a single query via email using just two nodes (Email and Run Query), sometimes you want to include more explanation or have multiple search results. The following flow shows an example of a more complex email:
```

Comment:
"sometimes you want"
just a suggestion but maybe should be: "sometimes you may want"